lib/seadsa/Graph.cc

```diff
@@ -398,9 +399,8 @@ const Cell &Node::getLink(Field _f) const {
   assert(!FieldType::IsNotTypeAware());
   assert(!f.hasOmniType());
   Field omni = f.mkOmniField();
-  if (m_links.count(omni)) return *m_links.at(omni);
-
-  assert(false); // unreachable
+  assert(m_links.count(omni));
```

Comment:
this change is important

Reply:
the diff is too confusing. You are replacing a return with assert. That cannot be correct.